src/braket/device_schema/jaqcd_device_action_properties.py

```diff
@@ -75,3 +77,4 @@ class JaqcdDeviceActionProperties(DeviceActionProperties):
 
     supportedOperations: List[str]
     supportedResultTypes: Optional[List[ResultType]]
+    explicitQubits: bool = False
```

Comment:
Should this be optional? For example for simulators, it would be False if there's no entry for this.

Reply:
I'm keeping this optional for backwards compatibility; once the feature is plumbed into the service and all devices populate this, then the default will be safe to remove, if we want to. As for simulators, our current simulators don't support non-contiguous qubit indices, so `False` is correct for them too.

Reply:
I mean optional as in explicitQubits: Optional[bool], so that the default is None instead of False, if there's no entry. 

Also, for clarity and to avoid the same name as in the IR, I'd suggest "explicitQubitsSupported" or "supportedExplicitQubits"

Reply:
Will make the name change.

As for the optional part, I'm getting [billion dollar mistake](https://www.infoq.com/presentations/Null-References-The-Billion-Dollar-Mistake-Tony-Hoare/) (but nowhere near as catastrophic) vibes. I feel like the law of excluded middle applies here; the feature is either supported or unsupported. There really isn't a need to have a special null value if all that means is that it's still unsupported.

Reply:
My main concern is that for the simulator, if there's no entry, it would be "False", which is wrong and confusing to users.

However, if we only merge this to main when there are entries, then it's fine.

Reply:
It should be false for the simulator though; explicit qubit allocation isn't supported.

Reply:
Oh, I think we have different definitions then for explicit qubit allocation. I thought it meant the absence of qubit rewiring, and there's no qubit rewiring for simulators, hence it allows explicit qubit allocation by default.